api/user/schema.py

```diff
@@ -175,24 +176,27 @@ def mutate(self, info, email, **kwargs):
         query_user = User.get_query(info)
         active_user = query_user.filter(UserModel.state == "active")
         exact_user = active_user.filter(UserModel.email == email).first()
+
         if not exact_user:
             raise GraphQLError("User not found")
 
         new_role = RoleModel.query.filter_by(id=kwargs['role_id']).first()
+
```

Comment:
@cop1fab please refer to comment on line 59 by @rajeman and ensure all unnecessary changes are fixed.